src/main/java/useCases/recipeBuilder.java

```diff
@@ -0,0 +1,18 @@
+package useCases;
+
+import entities.recipe;
+import entities.ingredient;
+
+import java.util.ArrayList;
+
+public class recipeBuilder {
+    /**
+     *recipeBuilder class, not to be instantiated.
+     *Handles the creation of recipes using API output.
+     *createRecipe takes input from collected API information and uses it to create a new recipe instance.
+     */
+    public static recipe createRecipe(String name,ArrayList<ingredient> ingList, int time, int price, String info){
+        recipe n = new recipe(name, ingList, time, price, info);
+        return n;
```

Comment:
Consider using a more meaningful variable name than n or consider doing 'return new recipe(name, ingList, time, price, info);'

Further, the abbreviation ingr might be better than ing so that we don't have any ambiguities. [Ingr](https://www.abbreviations.com/term/2201113)

Reply:
Unless everyone is good with using ing.